requirements.txt

```diff
@@ -19,6 +19,7 @@ idna==2.9                 # via requests
 importlib-metadata==1.5.0  # via argcomplete
 jmespath==0.9.4
 kappa==0.6.0
+pathspec>=0.7.0
```

Comment:
Looks like you need to add this dependency to `requirements.in` rather than this file, which is required for the auto generation of test_requirements.txt. That should fix the failing tests.

Reply:
Could you please check the updates and help me get this pull request into master?